src/cloudevent/mocks/alerts/performance-on-threshold-alert-published.ts

```diff
@@ -0,0 +1,61 @@
+import { DeepPartial, MockCloudEventAbstractFactory } from '../../types';
+import { CloudFunction } from 'firebase-functions/v2';
+import {
+  PerformanceEvent,
+  ThresholdAlertPayload
+} from 'firebase-functions/v2/alerts/performance';
+import {
+  getBaseCloudEvent,
+  getEventFilters,
+  getEventType,
+  PROJECT_ID,
+  APP_ID,
+} from '../helpers';
+import { FirebaseAlertData } from 'firebase-functions/v2/alerts';
+
+export const performanceThresholdOnThresholdAlertPublished: MockCloudEventAbstractFactory<PerformanceEvent<
+  ThresholdAlertPayload
+>> = {
+  generateMock(
+    cloudFunction: CloudFunction<PerformanceEvent<ThresholdAlertPayload>>
+  ): PerformanceEvent<ThresholdAlertPayload> {
+    const source = `//firebasealerts.googleapis.com/projects/${PROJECT_ID}`;
+    const alertType = 'performance.threshold';
+    const appId = APP_ID;
+    return {
+      ...getBaseCloudEvent(cloudFunction),
+      alertType,
+      appId,
+      source,
+      data: getThresholdAlertPayload(),
```

Comment:
Users should be able to override the data/payload using the `cloudEventPartial` (missing 2nd argument)

Reply:
Scratch that, merging happens automatically 👍